locale/shine/extensions/voterandom/ruRU.json

```diff
@@ -0,0 +1,36 @@
+{
+	"SHUFFLE_RANDOM": "Перемешать игроков случайным образом по командам.",
+	"SHUFFLE_SCORE": "Перемешать игроков основываясь на показателе очков в минуту.",
+	"SHUFFLE_KDR": "Перемешать игроков основываясь на KDR.",
+	"SHUFFLE_HIVE": "Перемешать игроков основываясь на рейтинге Hive.",
```

Comment:
These 4 messages at the top are used as parts of some of the messages below, so they shouldn't end with a `.`.

The `AUTO_SHUFFLE`, `PREVIOUS_VOTE_SHUFFLE`, `NEXT_ROUND_SHUFFLE`, `SHUFFLE_AND_RESTART` and `SHUFFLING_TEAMS` all have their `{ShuffleType}` bit replaced with one of these.